project/SalatBuild.scala

```diff
@@ -77,18 +77,19 @@ object BuildSettings {
     parallelExecution in Test := false,
     testFrameworks += TestFrameworks.Specs2,
     resolvers ++= Seq(typeSafeRepo, typeSafeSnapsRepo, oss, ossSnaps),
-    javacOptions ++= Seq("-source", "1.6", "-target", "1.6"), 
+    javacOptions ++= Seq("-source", "1.6", "-target", "1.6"),
     scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-language:_"),
     crossScalaVersions ++= Seq("2.10.6")
   )
 }
 
 object Scalariform {
 
-import com.typesafe.sbt.SbtScalariform._
-import scalariform.formatter.preferences._
+  import scalariform.formatter.preferences._
+  import com.typesafe.sbt.SbtScalariform
+  import com.typesafe.sbt.SbtScalariform.ScalariformKeys
 
-  val settings = scalariformSettings ++ Seq(
+  val settings = SbtScalariform.defaultScalariformSettings ++ Seq(
```

Comment:
Motivation:

> If you don't want sbt to automatically format your source files when the tasks compile or test:compile, just add defaultScalariformSettings instead of scalariformSettings to your build definition.

[https://github.com/sbt/sbt-scalariform](url)